### DISCUSSION TITLE
Changing code font size in template.pptx

### DISCUSSION THREAD

**Discussion by mjfrigaard** [davidgohel/officedown#108] [Q&A]
Hello everyone--I apologize if this question has been asked and answered. I'm trying to adjust the code font size (chunks and output) for a pptx template. I have the following file for a minimal example: 

![image](https://user-images.githubusercontent.com/8368326/234424509-f3f56a01-b780-45ec-8ffd-b4ff14086e00.png)

The body font is set to Ubuntu (size 28), and on the master I can see the same setting: 

<img width="1576" alt="image" src="https://user-images.githubusercontent.com/8368326/234424714-921f412a-1b89-4092-9ba2-6429ea538c69.png">

However, I also inserted a code text box in the pptx with the font style I'd like the code to formatted in when it's rendered from R Markdown: 

<img width="1529" alt="image" src="https://user-images.githubusercontent.com/8368326/234424928-bea293e6-ad6f-423b-9cdd-16f99faa5df2.png">

When I run the following R Markdown file (using my template): 

~~~markdown
---
title: "pptx from Rmd"
subtitle: "subtitle: pptx from Rmd"
output: 
  powerpoint_presentation:
    reference_doc: "reference_test.pptx"
---

```{r setup, include=FALSE}
library(knitr)
knitr::opts_chunk$set(echo = FALSE)
```

## Code font sizes in PPTX

The text body is Ubuntu (like the `reference_text.pptx`) size 28

<br>

`This inline code is Courier size 28`

<br>

```{r , echo=TRUE, eval=FALSE}
print("All the code block code is also Courier size 28")
```

<br>

```{r , echo=FALSE, eval=TRUE}
print("All the code output is also Courier size 28")
```
~~~

I get the following: 

<img width="1220" alt="image" src="https://user-images.githubusercontent.com/8368326/234425266-f6b4dc78-d36e-4ec4-8d28-b552164dc011.png">

**Where does this Courier font style come from, and can I change it for only code chunks**?

I've tried dropping this in the .Rmd (with no avail): 

```css
<style type="text/css">

body, td {
   font-size: 14px;
}
code.r{
  font-size: 20px;
}
pre {
  font-size: 20px
}
</style>
```

Any help would be greatly appreciated! 

**EDIT:** here is a GitHub repo with all the files: https://github.com/mjfrigaard/rmd2pptx

**Comment by davidgohel** [comment]
hello,

this out of the scope of the package "officedown". This is about rmarkdown.

But if it helps, this page explains how to customize powerpoint template for rmarkdown, https://ardata-fr.github.io/officeverse/office-documents-generation.html#powerpoint-presentation-properties. I am pretty sure 'posit' provides many ressources about this in their documentations.
